bin/init-docker.sh

```diff
@@ -0,0 +1,27 @@
+echo "setting up docker container. This can take a few minutes depending on your netowrk speed"
+
+# clone the data repository
+git clone https://github.com/Scripta-Qumranica-Electronica/Data-files.git ./tmp-data
+
+# cd into the directory
+cd ./tmp-data
+
+# Cleanup up any images that might exist if running setup a second time
+docker stop SQE_Database && docker rm SQE_Database
+
+# Build the image
+docker build -t sqe-maria:latest .
+
+# start the container
+docker run --name SQE_Database -e MYSQL_ROOT_PASSWORD=none -d -p 3307:3306 sqe-maria:latest
+
+# Wait a minute or so to ensure the container is started, and the DB process is initialized
+echo "waiting until container is ready ..."
+sleep 45s
+
+# ... then
+# import the data
+docker exec -i SQE_Database /tmp/import-docker.sh
+
+# cleanup
+cd .. && rm -Rf ./tmp-data
```

Comment:
This looks really nice!  Will you write a similar script to pull the SQE_DB_API into ./resources/perl-libs?  Also, perhaps at some later point these scripts should read the sqe-manifest.json file so that they pull down the right version of the database and SQE_DB_API.

Reply:
Yes, that's my thinking.

So you could pull any tag/branch, etc, and run npm run setup and be ready to go in a few minutes.

Reply:
Awesome!